### DISCUSSION TITLE
隔一段时间会退出qq登录

### DISCUSSION THREAD

**Discussion by niconieryr** [cinit/QAuxiliary#1230] [Q&A]
我的qq版本是9.1.5.20120，我在使用此模块的时候，qq总是会隔一段时间自动退出登录，且每次都要进行新设备登录验证。是不是我的qq版本太高了，还是说某些功能会导致这样。

**Comment by EatHatsuneShallots** [comment]
是所有设备都会退出登录并显示以下信息吗：
下线通知
你的账号当前登录已过期，请重新登录。

并且是否伴随每次退出登录后都得扫码登录电脑版 NTQQ 的问题

**Reply by niconieryr** [reply]
是的，我的平板和手机qq只要使用了模块就会隔段时间自己退出登录并显示你发的那条信息。但是不用扫码登录电脑qq，而是在当前设备的qq上重新进行新设备登录验证。

**Reply by EatHatsuneShallots** [reply]
> 我的qq只要使用了模块就会隔段时间自己退出登录

也许你应该尝试降级 QQ（不包括模块和框架）
并开启模块的 **禁用热补丁** 和 **拦截环境检测包** 功能

**Reply by Austra1** [reply]
我把qq降级到了8.9.50打开拦截也屏蔽了热更新也有这个问题

**Reply by EatHatsuneShallots** [reply]
> 我把qq降级到了8.9.50打开拦截也屏蔽了热更新也有这个问题

清除了热补丁吗，可能是你的账号被风控了

**Reply by evercatowo** [reply]
> 是的，我的平板和手机qq只要使用了模块就会隔段时间自己退出登录并显示你发的那条信息。但是不用扫码登录电脑qq，而是在当前设备的qq上重新进行新设备登录验证。

跟你一模一样
